ruby/ql/lib/codeql/ruby/security/BadTagFilterQuery.qll

```diff
@@ -65,7 +65,7 @@ predicate isBadRegexpFilter(HtmlMatchingRegExp regexp, string msg) {
   regexp.matches("<!-- foo --!>") and
   exists(int a, int b | a != b |
     regexp.fillsCaptureGroup("<!-- foo -->", a) and
-    // <!-- foo --> might be ambigously parsed (matching both capture groups), and that is ok here.
+    // <!-- foo --> might be ambiguous parsed (matching both capture groups), and that is ok here.
```

Comment:
This change would be grammatically incorrect. 

```suggestion
    // <!-- foo --> might be ambiguously parsed (matching both capture groups), and that is ok here.
```
or perhaps
```suggestion
    // <!-- foo --> might be parsed ambiguously (matching both capture groups), and that is ok here.
```

Reply:
Sigh. I saw that in one place and I know I switched it to the `ly` form.

Reply:
This file is shared with javascript and python, so you might have seen it there.